entities.py

```diff
@@ -101,6 +99,24 @@ def move_towards(self, dest):
         if self.game_map.walkable[next_tile] and not self.game_map.get_blocking_entity_at_location(next_tile):
             self.move(direction)
 
+    def place(self, game_map, position):
```

Comment:
I don't know how I feel about this, IMO this should be either in the registry or in the dungeon code

Reply:
OK, I'll put it in the dungeon then which also makes sense. Having the registry handle entity placement doesn't make much sense to me.